### PR TITLE
link character request to its page

### PR DESCRIPTION
**Pull request by clemgi0**

### CODE REVIEW

File: src/models/request/character-request.ts [hunk 1]
```diff
@@ -45,18 +44,18 @@ export class CharacterRequest extends Request<CharacterResult[]> {
...
-                OPTIONAL { <${selectedCharacterUri}> dbp:siblings ?siblings }
+                OPTIONAL { dbr:${selectedCharacterUri} dbo:thumbnail ?thumbnail }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:abode ?abode }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:children ?children}
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:consort ?consort }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:godOf ?godOf }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:parents ?parents }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:planet ?planet }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:symbol ?symbol }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:siblings ?siblings }
```
Comment (cfstcyr) [inline src/models/request/character-request.ts:58]: look for fix in case of special characters

File: src/models/request/search.ts [hunk 2]
```diff
@@ -33,3 +27,19 @@ export interface LocationResult {
...
+
+export interface CharacterResult {
+    url: string;
+    label: string;
+    uriPart: string;
+    abstract: string;
+    thumbnail: string | null;
+    abode: string | null;
+    children: string[] | null;
+    consort: string | null;
```
Comment (cfstcyr) [inline src/models/request/search.ts:39]: i think consort should be an array

File: src/pages/character-page/character-page.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    }>();
+
+    useEffect(() => {
+        const characterName = decodeURIComponent(encodedCharacterName || "");
+
+        const characterRequest = new CharacterRequest(characterName);
+
+        characterRequest.execute().then((results) => {
+            setCharacterResult(results);
+        });
```
Comment (cfstcyr) [inline src/pages/character-page/character-page.tsx:21]: should handle error (display "Error" on the page).

**Review by cfstcyr — DISMISSED** [review]

File: src/models/request/character-request.ts (hunk 1 above)
Comment (clemgi0) [reply src/models/request/character-request.ts]: changed

File: src/pages/character-page/character-page.tsx (hunk 3 above)
Comment (clemgi0) [reply src/pages/character-page/character-page.tsx:21]: add an error handler

File: src/models/request/search.ts (hunk 2 above)
Comment (clemgi0) [reply src/models/request/search.ts:39]: done

**Review by josue-venegas-almonacid — APPROVED** [review]

**Pull request merged**